mkdocs.yml

```diff
@@ -22,14 +22,13 @@ nav:
   - Installation: installing.md
   - Getting Started: getting-started.md
   - Managing test scripts:
-      - Creating: scripts-creating.md
-      - Running: scripts-running.md
-      - Getting results: scripts-getting-results.md
-  - Executors:
       - Postman Collections: executor-postman.md
       - Cypress Tests: executor-cypress.md
       - Curl Commands: executor-curl.md
-      - Creating your own: executor-custom.md
+      - Running: scripts-running.md
```

Comment:
I would move it up as it's some kind of introduction w/o details - and now we're starting from postman which is deep dive into postman collections runner

Reply:
I have moved it 
```
  - Managing test scripts:
      - Running: scripts-running.md
      - Getting results: scripts-getting-results.md
  - Executors:
      - Postman Collections: executor-postman.md
      - Cypress Tests: executor-cypress.md
      - Curl Commands: executor-curl.md
      - Creating custom executor: executor-custom.md
```

Reply:
I moved it around please take another look